doc/_i18n/jp/tutorials/recipes/fsm.md

```diff
@@ -13,6 +13,8 @@ mc\_rtcのコントローラーでは、状態遷移図の概念を実装でき
 
 状態の実装については後ほど説明します。ここでは、状態に関する4つの主なメソッドについて説明します。
 
+{% comment %} FIXME Out-of-date translation for configure {% endcomment %}
```

Comment:
I can update Japanese sentence in this part if necessary.

Reply:
Thanks @mmurooka that would be very nice! (if you don't mind, there's also two new missing translations in #237)

Reply:
I made two PRs https://github.com/gergondet/mc_rtc/pull/8 https://github.com/gergondet/mc_rtc/pull/9
(I favor the multi-language tutorials, but the Japanese text in the tutorials was quite difficult for me to read...)